app/views/liquidum/conversions/index.html.slim

```diff
@@ -1,5 +1,5 @@
 = sts.form_for(@conversion, url: conversions_path, data: { turbo: false }) do |f|
-  = sts.card(title: 'Conversions') do |card|
+  = sts.card :conversions do |card|
```

Comment:
I guess we need to add translations for this else it will give translation missing.
same for below 2.